tensorflow/java/src/main/java/org/tensorflow/Server.java

```diff
@@ -0,0 +1,82 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+package org.tensorflow;
+
+/**
+ * An in-process TensorFlow server, for use in distributed training.
+ *
+ * A {@code tf.train.Server} instance encapsulates a set of devices and a
+ * {@code tf.Session} target that can participate in distributed training. A
+ * server belongs to a cluster (specified by a {@code tf.train.ClusterSpec}),
+ * and corresponds to a particular task in a named job. The server can
+ * communicate with any other server in the same cluster.
```

Comment:
Let's add some commentary/sample code on how to use objects of this class (`start()`, `stop()` etc.)